java-checks-test-sources/src/main/files/non-compiling/checks/SimpleStringLiteralForSingleLineStringsCheck.java

```diff
@@ -0,0 +1,19 @@
+package checks;
+
+public class SimpleStringLiteralForSingleLineStringsCheck {
+  
+  public void str() { // Noncompliant@+1 [sc=23;endColumn=42]]{{Use simple literal for a single-line string.}}
+    String question = """
+              What's the point, really?""";
+
+    String question1 = "What's the point, really?"; // Compliant
+
+    String question2 = """
+              What's the point, really
+              ?"""; // Compliant, 3 lines
```

Comment:
This is 3 lines for our engine (when you call `textBlock.value()`) but it represents a two lines string in java, right?

Reply:
yes. It's actually 2 lines, because it's not possible to write `"""What's the point, really?"""`